stonk_market_mayhem.py

```
@@ -0,0 +1,89 @@
+
+portfolio = {}
+money = 10000
+
+def show_menu() -> None:
+    """Display the main menu options to the user."""
+    print("1. View Stocks")
+    print("2. Buy Stock")
+    print("3. Sell Stock")
+    print("4. View Portfolio")
+    print("5. Exit")
+def view_stocks():
```

Ruff (0.8.2)

Missing return type annotation for public function `view_stocks`

Add return type annotation: `None`

(ANN201)